cmd/host/src/lib.rs

```diff
@@ -234,7 +234,9 @@ fn print_panic(d: Vec<u8>) -> Result<()> {
     println!("thread:    {:#x}", data.thread);
 
     if let Some(time) = data.time {
-        let dt: DateTime<Utc> = time.system_time.into();
+        let dt = DateTime::from_timestamp(time.sec as i64, time.nsec)
+            .ok_or_else(|| anyhow!("invalid timestamp"))?;
```

Comment:
not a big deal, but, in case this ever actually happens, would it maybe be better to just print "invalid timestamp" in the output and keep going so that we can display the rest of the panic data?

currently, if someone is on Windows and tries to read a panic with a timestamp that Windows can't understand, we'll bail out without printing registers, stack, etc. it seems like it might be nicer to just say "hey, we couldn't represent this timestamp as RFC3339 on your system for some reason, but here's the raw seconds/nanoseconds, and here's the rest of your dump"...

Reply:
Good call; will fix!